src/tests/ftest/aggregation/aggregation_punching.yaml

```diff
@@ -0,0 +1,39 @@
+hosts:
+  test_servers:
+    - server-A
+    - server-B
+  test_clients:
+    - client-A
+    - client-B
+timeout: 600
+server_config:
+    name: daos_server
+    servers:
+        log_mask: INFO
+        bdev_class: nvme
+        bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+pool:
+    mode: 146 # 146 is RW
+    name: daos_server
+    scm_size: 8000000000
+    nvme_size: 80000000000
+    svcn: 1
+    control_method: dmg
+container:
+    type: POSIX
+    control_method: daos
+mdtest:
+  client_processes:
+      np: 2
+  test_dir: "/"
+  dfs_destroy: False
+  iteration: 1
+  manager: "MPICH"
+  mdtest_params:
```

Comment:
Optional: this can be removed and the params under this can be directly under `mdtest:`